elf_reader.go

```diff
@@ -1134,7 +1132,7 @@ func (ec *elfCode) loadKconfigSection() error {
 	}
 
 	var ds *btf.Datasec
-	err := ec.btf.TypeByName(kconfigMap, &ds)
+	err := ec.btf.TypeByName(".kconfig", &ds)
```

Comment:
It's only used in three places now: here in the elf reader and when fixing up datasec in btf. The one use for MapSpec.Name is purely informational, so if we get it wrong it doesn't matter much.

In general I'm not a big fan of turning string literals into a constant, since it adds a layer of indirection. When reading code it's not clear that it's a constant (could be a var) and also what it's value is. If a string literal is repeated over an over as a key I try to reduce the number of lookups instead of introducing a constant.